sensor_msgs/PointCloud2[] cones_clouds
---
uint8[] colors